### DISCUSSION TITLE
MScAc%40UofT/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#117] [Announcements]
# MScAc%40UofT/

https://csmsapp.org/MScAc%40UofT/

**Comment by Reed-yang** [comment]
- 只填大三一年？还是大三时的三年gpa？
- 有无UoT的mscs录取信息(´;︵;`)

**Reply by xichenpan** [reply]
前者
陆本基本没有录取，可以申phd

**Comment by lorenz-liu** [comment]
这个面试的通过率可以参考一下吗？

**Reply by xichenpan** [reply]
第一批发的基本全过，越往后发的过率越低

**Reply by xichenpan** [reply]
一般一个月一批面试

**Comment by ErwinZhou** [comment]
求问2024Fall是否还存在对陆本和海本一视同仁的情况呢
